### ISSUE TITLE
Packages should declare all deps and peer deps in package.json

### ISSUE THREAD

**Issue by jaydenseric** [apollographql/react-apollo#3161]
**Intended outcome:**

Concerning [`@apollo/react-hooks`](https://unpkg.com/@apollo/react-hooks@0.1.0-beta.7/) (and other packages that may have the same issue), all `dependencies` and `peerDependencies` should be declared in `package.json`. This allows tools such as [Bundlephobia](https://bundlephobia.com) to work.

**Actual outcome:**

[`@apollo/react-hooks`](https://unpkg.com/@apollo/react-hooks@0.1.0-beta.7/) does not declare `fast-json-stable-stringify` and `react-dom/server` in dependencies or peer dependencies in `package.json`. This prevents tools such as [Bundlephobia](https://bundlephobia.com) from working.

**How to reproduce the issue:**

https://bundlephobia.com/result?p=@apollo/react-hooks@0.1.0-beta.7

<img width="811" alt="Screen Shot 2019-06-24 at 9 21 15 am" src="https://user-images.githubusercontent.com/1754873/59983333-ae31bc80-9661-11e9-9eeb-31be6bfe6f87.png">

**Version**

- @apollo/react-hooks@0.1.0-beta.7

**Comment by jaydenseric** [comment]
I've just realised the `latest` release on npm is a few versions behind the actual latest 🤨

<img width="307" alt="Screen Shot 2019-06-24 at 9 34 21 am" src="https://user-images.githubusercontent.com/1754873/59983464-4b412500-9663-11e9-96a3-686616362ae9.png">

Still has the issue tho:

https://bundlephobia.com/result?p=@apollo/react-hooks@0.1.0-beta.9

**Comment by hwillson** [comment]
Thanks for reporting this @jaydenseric. I've fixed the `latest` issue, and will look into get the deps straightened out so bundlephobia works properly. I did push a new version earlier today that should help with the missing `react-dom` peer dep, but BP seems to be taking a while to pick it up:

![Screenshot 2019-06-24 10 55 16](https://user-images.githubusercontent.com/137740/60029137-914fb600-966e-11e9-8d7c-f75ee42e1dd8.png)

I guess whatever cache it's using hasn't been updated. I'll check back on things in a bit.
